doc/faq.rst

```diff
@@ -133,6 +133,26 @@ configuration options, e.g. to specify a custom location for storing the
 datasets.
 
 
+A function uses multiple CPU cores even though I didn't tell it to. Why?
+------------------------------------------------------------------------
+Ordinarily in MNE-python the `parallel` module is used to deploy multiple
+cores via the `n_jobs` variable. However, functions like
```

Comment:
double-backticks on code. Singe-backticks will try to link. We actually could get `parallel` to link with
```
:mod:`joblib`
```
if we add it to the known projects. But that's getting deep into bonus territory :)